graylog2-web-interface/src/components/common/Spinner.test.jsx

```diff
@@ -35,4 +35,9 @@ describe('<Spinner />', () => {
 
     expect(container.firstChild).toHaveStyle('visibility: visible');
   });
+
+  it('should forward its props to its icon', () => {
+    const { getByTestId } = render(<Spinner data-testid="icon-test-id" />);
+    expect(getByTestId('icon-test-id')).not.toBeNull();
+  });
```

Comment:
This was the easies way I could find to test the described functionality. Removing the `...rest` usage results in a test failure.